addonscript/requirements/base.py37.txt

```diff
@@ -62,9 +62,9 @@ async-timeout==3.0.1 \
     #   -r requirements/base.in
     #   aiohttp
     #   taskcluster
-attrs==20.3.0 \
-    --hash=sha256:31b2eced602aa8423c2aea9c76a724617ed67cf9513173fd3a4f03e3a929c7e6 \
-    --hash=sha256:832aa3cde19744e49938b91fea06d69ecb9e649c93ba974535d08ad92164f700
+attrs==21.2.0 \
```

Comment:
Quite a lot's going on [here](https://www.attrs.org/en/stable/changelog.html#id1) but it looks like the only backwards-incompatible stuff is for py34 and between 21.1.0 -> 21.2.0 ?